app/javascript/packs/homePage.jsx

```diff
@@ -1,7 +1,19 @@
 import { h, render } from 'preact';
+import { renderFeed } from './homePageFeed';
 
 /* global userData */
 
+// This logic is similar to that in initScrolling.js.erb
+// that prevents the classic Algolia scrolling for the front page.
+const frontPageFeedPathNames = new Map([
```

Comment:
This is the best way for now that I could come up with for updating the feed URL when we grab article data.